docs/root/version_history/v1.20.7.rst

```diff
@@ -0,0 +1,26 @@
+1.20.7 (July 21, 2022)
+======================
+
+Incompatible Behavior Changes
+-----------------------------
+*Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+
+Minor Behavior Changes
+----------------------
+*Changes that may cause incompatibilities for some users, but should not for most*
+
+Bug Fixes
+---------
+*Changes expected to improve the state of the world and are unlikely to have negative effects*
+
+* ci: fix disk space issue that have prevented publication.
```

Comment:
this doesnt seem to match the changelog that it was copied from

Reply:
Thank you for catching that. I was tired and it seems I copied https://raw.githubusercontent.com/envoyproxy/envoy/release/v1.20/docs/root/version_history/v1.20.6.rst where I ought to have copied https://github.com/envoyproxy/envoy/blob/release/v1.20/docs/root/version_history/current.rst

I will amend & force push so we keep two commits.

Reply:
So I pushed the change to amend this, and as a bonus CI turned green 🥳 
Ready for another pass